docs/Node Operators/running-a-chainlink-node.md

```diff
@@ -124,49 +127,33 @@ GAS_UPDATER_ENABLED=true
 ALLOW_ORIGINS=*" > ~/.chainlink/.env
 ```
 
-### Set your Ethereum Client URL
```

Comment:
This change doesn't seem clear to me. 

Do we have to do environment variables for the ETH_URL? Do we think this is better?

Reply:
> Do we have to do environment variables for the ETH_URL? Do we think this is better?

We could provide a config template and have users fill it out manually. That's what I ended up doing when I first tried this doc because the env variables didn't work as documented.

Env variables are easier for some users because copying and pasting commands is straightforward as long as the commands do exactly what we expect. My personal bias is towards manually editing a config template because that's what is fastest for me usually.

The change I made here is to fix the broken `docker inspect` command and combine URL configuration into one section with sub-options. Setting the URL is the task. Setting up the ETH client is a prerequisite at the beginning of the page.

Reply:
@PatrickAlphaC I revised this section to remove the environment variable complexity. It now asks users to create a `.env` file and edit the values. https://deploy-preview-192--dreamy-villani-0e9e5c.netlify.app/docs/running-a-chainlink-node/#create-an-environment-file